src/lib/loaders/loaders_without_authentication/gravity.js

```diff
@@ -45,6 +45,7 @@ export default requestIDs => {
     ),
     saleArtworkRootLoader: gravityLoader(id => `sale_artwork/${id}`),
     saleArtworksLoader: gravityLoader(id => `sale/${id}/sale_artworks`),
+    saleArtworksAllLoader: gravityLoader("sale_artworks"),
```

Comment:
How come you need both authenticated and unauthenticated loaders? Doesn't this feature only work for logged-in users?

Reply:
I spoke with @ashkan18 over slack, but the idea is to be able to do 
```
{
  sale_artworks(live: true) {
    ...
  }  
}
```
Which doesn't need authentication. However, when using 
```
{
  sale_artworks(live: true, include_artworks_by_followed_artists: true) {
    ...
  }  
}
```
It uses the new `/sale_artworks` endpoint and requires auth.

Reply:
is this something we need to support now? if there is no current use case for it i'd defer supporting only filter by `live_sale=true` to later. This can potentially return many artworks and even paginated the order is going to be somewhat random.

Reply:
@ashkan18 - Definitely not needed for now! We can loop back once the new `/auctions` work starts.